### PR TITLE
Explain environment variables

### LINKED ISSUE #160: Complete documentation: Describe each environment variable
Hello folks,

I think it'd be great if every `ENV VAR` used in the images would  be explained in the `README.md`, and also it can be included in the `Wazuh official documentation` once they're ready.
 
Feel free to share your thoughts on this here.

Regards

### PR DESCRIPTION
**Pull request by xr09**
This PR fixes #160

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,13 +22,71 @@ In addition, a docker-compose file is provided to launch the containers mentione
...
+ELASTIC_CLUSTER="false"					# Setup a cluster
+CLUSTER_NAME="wazuh"					# Cluster name
+CLUSTER_NODE_MASTER="false"				# Set node as master
+CLUSTER_NODE_DATA="true"				# Store data on this node
+CLUSTER_NODE_INGEST="true"				# Setup as ingest node
+CLUSTER_NODE_NAME="wazuh-elasticsearch"                 # Name for this node
+CLUSTER_MASTER_NODE_NAME="master-node"			# Name of the master node
+CLUSTER_MEMORY_LOCK="true"				# Set Elasticsearch memory lock
+CLUSTER_DISCOVERY_SERVICE="wazuh-elasticsearch"         # Set discovery service
+CLUSTER_NUMBER_OF_MASTERS="2"				# Nomber of masters on the cluster
```
Comment (Cptbogo) [inline README.md:54]: Line 54 description typo; should be "# Number of masters on the cluster"

Comment (xr09) [reply README.md]: Fixed in 7a70334. Good catch!

**Comment by xr09** [comment]
Removed Elasticsearch section since starting with 4.0 release we'll stop re-building Elasticsearch Docker image and will use the original instead.

**Review by Zenidd — CHANGES_REQUESTED** [review]
Showing the default values for the environment vars could be a good point to help users choosing their configuration.

**Comment by xr09** [comment]
> Showing the default values for the environment vars could be a good point to help users choosing their configuration.

I'm on it :+1:

**Review by manuasir — APPROVED** [review]

**Pull request merged**